code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -182,13 +182,12 @@
 
 /obj/item/projectile/bullet/sniper/accelerator/Range()
 	..()
-	damage += 5
-	if(damage > 40)
-		icon_state = "gaussstrong"
+	damage += 7
```

Comment:
This is also affecting the balance of the original projectile, file from the nuke op sniper rifle. Was this your intent?